CRM/Core/Payment/Manual.php

```diff
@@ -96,7 +96,7 @@ public function getPaymentFormFields() {
       // Really we should render check_number here, but we need to review how we edit
```

Comment:
Actually that change is doing the reverse of what you say Joe & I think it may be a mistake / in conflict with changes & approach in other PRs